nesta/packages/dq/text.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
+import itertools
+from collections import Counter
+##string length
+def string_length(input):#, column):
+    '''string_length
+    Lengths of strings.
+
+    Args:
+        input (:obj:`iter` of :obj:`str`): A sequence of string objects.
+
+
+    Returns:
+        flat_length_list (:obj:`list`): A numpy array of string lengths.
+
+    '''
+    series = pd.Series(input)
+
+    flat_length_list = series[~pd.isnull(series)].str.len().values
+
+    return flat_length_list
+
+
+def string_counter(input):
+
+    '''string_counter
+
+    Args:
+        input (:obj:`iter` of :obj:`list`): A sequence of string objects.
+
+    Returns:
+        count (:obj:`Counter`): Counter object.
+    '''
+
+    series = pd.Series(input)

```

Comment:
`input` is a built in Python function. Generally best practice not to overwrite them.